### PR TITLE
Add Sony Xperia Z5 Compact (suzuran)

### PR DESCRIPTION
**Pull request by IGNNE**
Should work with other Sony Z5* (kitakami) phones, just replace the codenames

### CODE REVIEW

File: manifests/sony_suzuran.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+     <remote  name="olefb"
+          fetch="https://github.com/olefb"
+          revision="cm-14.1" />
+
+     <project path="device/sony/suzuran" name="android_device_sony_suzuran" remote="los" />
+     <project path="device/sony/kitakami-common" name="android_device_sony_kitakami-common" remote="los" />
+     <project path="device/sony/common" name="android_device_sony_common" remote="los" />
+
+     <project path="kernel/sony/msm8994" name="android_kernel_sony_msm8994" remote="los" />
```
Comment (jbruechert) [inline manifests/sony_suzuran.xml]: Please use a fork of the kernel repository with the halium config changes included.

Comment (deraol) [reply manifests/sony_suzuran.xml]: What fork? I installed and did not start...

Comment (IGNNE) [reply manifests/sony_suzuran.xml]: Your email reached me at one of the rare times I have nothing to do, so I'll try to do it properly this time. Just a sec.

**Comment by APokorny** [comment]
@IGNNE could you have a look at the device manifest that was merged for suzuran

**Comment by IGNNE** [comment]
@APokorny The manifest looks good, but I have used a slightly different kernel config. Mine 'works', although I admit I have no idea what I did there ;-)
Have you tried to boot your kernel config?

**Comment by APokorny** [comment]
Only on the normal z5 (sumire). But since everything else is more or less identical I also submitted the other kitakamis. The result looks like that then: https://www.phoronix.com/scan.php?page=news_item&px=Mir-Back-On-Phones-UBports

Disclaimer :  I have stopped using hybris boot for some time, the fixup mount points script needs to be adapted. Which isnt a problem if you use halium boot. But it would be interesting to see if the data partition has a different number or device path in your case.

**Comment by IGNNE** [comment]
Nice! I will build it this evening, if it boots, I am happy. I experimented a bit with X11 and Ubuntu desktop etc., so I probably enabled lots of stuff that just 'looked right'. Great news!

**Comment by lnjX** [comment]
I'll close this, because it's a duplicate. If there are any changes needed to the current manifest open a new pull request.

**Pull request closed without merging**